src/components/Users/Users.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import * as axios from "axios";
+
+import photo000 from "../../assets/images/photo000.png";
+
+import s from "./Users.module.css";
+
+class Users extends React.Component {
+  componentDidMount() {
+    axios
+      .get("https://social-network.samuraijs.com/api/1.0/users")
+      .then((response) => {
+        this.props.setUsers(response.data.items);
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        {this.props.users.map((u) => (
+          <div key={u.id}>
+            <span>
+              <div>
+                <img
+                  src={u.photos.small != null ? u.photos.small : photo000}
+                  className={s.avatar}
+                />
+              </div>
+              <div>
+                {u.followed ? (
+                  <button
+                    onClick={() => {
```

Comment:
Consider creating a separate function to hold this handler logic.

Reply:
done